kotlin/src/main/kotlin/org/jetbrains/research/testspark/kotlin/KotlinPsiClassWrapper.kt

```diff
@@ -19,18 +18,18 @@ import org.jetbrains.kotlin.resolve.DescriptorToSourceUtils
 import org.jetbrains.research.testspark.core.data.ClassType
 import org.jetbrains.research.testspark.core.utils.importPattern
 import org.jetbrains.research.testspark.core.utils.packagePattern
-import org.jetbrains.research.testspark.helpers.psi.PsiClassWrapper
-import org.jetbrains.research.testspark.helpers.psi.PsiMethodWrapper
+import org.jetbrains.research.testspark.langwrappers.PsiClassWrapper
+import org.jetbrains.research.testspark.langwrappers.PsiMethodWrapper
 
 class KotlinPsiClassWrapper(private val psiClass: KtClassOrObject) : PsiClassWrapper {
     override val name: String get() = psiClass.name ?: ""
 
-    override val qualifiedName: String get() = psiClass.fqName!!.asString() ?: ""
+    override val qualifiedName: String get() = psiClass.fqName?.asString() ?: ""
```

Comment:
I remember we discussed not tolerating empty, fully qualified names but checking whether they are null prior he instantiation of `KotlinPsiClassWrapper`? 

Unfortunately, I could not find it. What was the decision, and will it be reflected as an issue?

Reply:
You are totally correct. It is my bad merge